src/pytest_springfield/README.md

```diff
@@ -0,0 +1,6 @@
+`dottedname` Module
+===================
+
+This module exists as a dummy module to help test imports of
+entity classes when using dotted-name strings for `EntityField`
+types.
```

Comment:
I figured it out, I just need to drop `__init__.py` in tests dir and it will work out.